projects/power_distribution/src/main.c

```diff
@@ -84,6 +85,7 @@ int main(void) {
   interrupt_init();
   soft_timer_init();
   gpio_init();
+  gpio_it_init();
```

Comment:
This will most likely need to be moved to the bottom of the setup block

Reply:
Why does it need to be moved?